docs/source/customizations/rhai-api.mdx

```diff
@@ -11,15 +11,15 @@ This article documents symbols and behaviors that are specific to [Rhai customiz
 
 Your Rhai script's [main file](./rhai/#main-file) hooks into the individual services of the Apollo Router's [request-handling pipeline](./overview/#how-customizations-work). To do so, it defines whichever combination of the following entry point hooks it requires:
 
-```javascript
+```rhai
```

Comment:
Our code-highlighting library Prism doesn't actually support Rhai yet, we've just pseudosupport that lets us label codeblocks as Rhai but still use javascript syntax parsing